src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -489,8 +507,9 @@ public static void GetFullPath_Windows_AlternateDataStreamsNotSupported()
         public static void GetFullPath_Windows_URIFormatNotSupported()
         {
             // Throws via our invalid colon filtering
-            Assert.Throws<NotSupportedException>(() => Path.GetFullPath("http://www.microsoft.com"));
-            Assert.Throws<NotSupportedException>(() => Path.GetFullPath("file://www.microsoft.com"));
+            // Full Framework TFM < 4.6.2 throws ArgumentException. .Net Core and Full Framework TFM >= 4.6.2 throws NotSupportedException
```

Comment:
We **don't** want that. It is part of the new normalization that we ignore this now. Desktop had the same change under quirk (there are many more URIs that we used to let through- it was sort of pointless).